interpreter/attributes.go

```diff
@@ -650,6 +707,11 @@ func (q *stringQualifier) Value() ref.Val {
 	return q.celValue
 }
 
+// Cost implements the Coster interface method.
+func (q *stringQualifier) Cost() (min, max int64) {
```

Comment:
Perhaps mention, `Cost returns zero for constant field qualifiers`

Reply:
done